style/Earth.css

```diff
@@ -731,7 +731,6 @@ MAIN POSTS
 	float: right;
 	margin-right: -218px;
 	position: relative;
-	text-align: left;
```

Comment:
So these rules are superfluous in the default, left-to-right layout?

Reply:
There shouldn't be any absolute left or right unless you are sure that regardless of the text direction, the text should remain on left or right.

If you need to mention the alignment, use start|end instead of left|right. I removed it as I found it unnecessary.